config/machine.cfg

```diff
@@ -10,7 +10,8 @@ square_corner_velocity: 5.0
 path: ~/printer_data/gcodes
 on_error_gcode:
     _PROBE_ON_ERROR_ACTION
-
+   status_leds COLOR=error
```

Comment:
Can you change "status_leds" to uppercase to follow the Gcode ISO norm ?

Also it's probably better to change the color of the leds before calling the `_PROBE_ON_ERROR_ACTION` so I would prefer to invert the two lines if possible.

Reply:
ok done